extensions/helix/package.build.ts

```diff
@@ -0,0 +1,217 @@
+// Save to package.json
+// ============================================================================
+
+import { Builder, generateIgnoredKeybinds } from "../../meta";
+import * as fs from "fs/promises";
+import { SelectionBehavior } from "../../src/api";
+
+const version = "0.1.0",
+      preRelease = 1,
+      preReleaseVersion = version.replace(/\d+$/, "$&" + preRelease.toString().padStart(3, "0"));
+
+export const pkg = (modules: Builder.ParsedModule[]) => ({
+
+  // Common package.json properties.
+  // ==========================================================================
+
+  name: "dance-helix-keybinds",
+  description: "Helix keybindings for dance",
+  version,
+  license: "ISC",
+
+  author: {
+    name: "Grégoire Geis",
+    email: "opensource@gregoirege.is",
+  },
+
+  repository: {
+    type: "git",
+    url: "https://github.com/71/dance.git",
+  },
+
+  engines: {
+    vscode: "^1.63.0",
+  },
+
+  displayName: "Dance",
+  publisher: "gregoire",
+  categories: ["Keymaps", "Other"],
+  readme: "README.md",
+  icon: "dance.png",
+
+  scripts: {
+    "package": "vsce package --allow-star-activation",
+    "publish": "vsce publish --allow-star-activation",
+    "package:pre": `vsce package --allow-star-activation --pre-release --no-git-tag-version --no-update-package-json ${preReleaseVersion}`,
+    "publish:pre": `vsce publish --allow-star-activation --pre-release --no-git-tag-version --no-update-package-json ${preReleaseVersion}`,
+  },
+
+  contributes: {
+    configurationDefaults: {
+      "dance.defaultMode": "helix/normal",
+      "dance.modes": {
+        "": {
+          hiddenSelectionsIndicatorsDecoration: {
+            after: {
+              color: "$list.warningForeground",
+            },
+            backgroundColor: "$inputValidation.warningBackground",
+            borderColor: "$inputValidation.warningBorder",
+            borderStyle: "solid",
+            borderWidth: "1px",
+            isWholeLine: true,
+          },
+        },
+        "input": {
+          cursorStyle: "underline-thin",
+        },
```

Comment:
If these two entries are omitted does Dance stop working normally?

Reply:
From quick testing, it doesn't, I just copy pasted the base settings because I wasn't sure what to with those.